packages/react-native-cli/src/commands/AutomateSymbolicationCommand.ts

```diff
@@ -26,45 +26,25 @@ const HERMES_INSTRUCTIONS = `You are running a version of React Native that we c
 
 `
 
-const BUGSNAG_CLI_INSTRUCTIONS = `bugsnag:upload-android task added to your package.json - run this task to upload Android source maps after a build.
+const BUGSNAG_CLI_INSTRUCTIONS = `The following tasks have been added to your package.json and can be run after a build to upload source maps to BugSnag:
+
+    bugsnag:upload-android
+    bugsnag:upload-ios
```

Comment:
Needs an update with the other commands - maybe also a few words against each to explain them?